prow/config.yaml

```diff
@@ -112,6 +112,49 @@ presubmits:
       - name: cache-ssd
         hostPath:
           path: /mnt/disks/ssd0
+    run_after_success:		
+    - name: pull-kubernetes-e2e-gce-canary		
+      agent: kubernetes		
+      always_run: true		
+      context: pull-kubernetes-e2e-gce-canary		
+      rerun_command: "/test pull-kubernetes-e2e-gce-canary"		
+      trigger: "(?m)^/test( all| pull-kubernetes-e2e-gce-canary),?(\\s+|$)"
+      spec:
+        containers:
+        - args:
+          - --bare
+          - --timeout=75
+          - "--clean"
+          - "--git-cache=/root/.cache/git"
+          - "--job=$(JOB_NAME)"
+          - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+          - "--service-account=/etc/service-account/service-account.json"
+          - "--upload=gs://kubernetes-jenkins/pr-logs"
+          env:
+          - name: GOOGLE_APPLICATION_CREDENTIALS
+            value: /etc/service-account/service-account.json
+          - name: USER
+            value: prow
+          - name: JENKINS_GCE_SSH_PRIVATE_KEY_FILE
+            value: /etc/ssh-key-secret/ssh-private
+          - name: JENKINS_GCE_SSH_PUBLIC_KEY_FILE
+            value: /etc/ssh-key-secret/ssh-public
+          image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170829-89d24535
+          volumeMounts:
+          - mountPath: /etc/service-account
+            name: service
+            readOnly: true
+          - mountPath: /etc/ssh-key-secret
+            name: ssh
+            readOnly: true
+        volumes:
+        - name: service
+          secret:
+            secretName: service-account
+        - name: ssh
+          secret:
+            defaultMode: 256
+            secretName: ssh-key-secret
```

Comment:
NOTE: this spec was made from a normal prow e2e job + the bazel build args from @krzyzacy's advice, I'm not completely sure if this is correct. PTAL